File: raptorsRobotsApp/src/main/java/pl/raptors/raptorsRobotsApp/domain/healthz/JVMData.java
package pl.raptors.raptorsRobotsApp.domain.healthz;

import lombok.Data;

@Data
public class JVMData {
    private String freeMemory;
    private String allocatedMemory;
    private String maxMemory;
    private String totalFreeMemory;

    public JVMData(long freeMemory, long allocatedMemory, long maxMemory, long totalFreeMemory) {
        this.freeMemory = String.valueOf(freeMemory);
        this.allocatedMemory = String.valueOf(allocatedMemory);
        this.maxMemory = String.valueOf(maxMemory);
        this.totalFreeMemory = String.valueOf(totalFreeMemory);
    }
}

File: angularclient/src/app/model/JVMdata.ts
export class JVMdata {
  freeMemory: string;
  allocatedMemory: string;
  maxMemory: string;
  totalFreeMemory: string;


  constructor(freeMemory: string = '', allocatedMemory: string = '', maxMemory: string = '', totalFreeMemory: string = '') {
    this.freeMemory = freeMemory;
    this.allocatedMemory = allocatedMemory;
    this.maxMemory = maxMemory;
    this.totalFreeMemory = totalFreeMemory;
  }}

